histoqc/wsihandles/WSIImageHandle.py

```diff
@@ -1,7 +1,17 @@
-import os
 from abc import ABC, abstractmethod
-# from .DicomHandle import DicomHandle
-# from .OpenSlideHandle import OpenSlideHandle
+from typing import Union
+from enum import Enum
+from pathlib import Path
+import os, logging
+
+
+class WSIHandles(Enum):
+    OPENSLIDE = 'openslide'
+    WSIDICOM = 'dicom'
+
+class WSIFileExtension(Enum):
+    OPENSLIDE = ['.bif', '.mrxs', '.ndpi', '.scn', '.svs', '.svslide', '.tif', '.tiff', '.vms', '.vmu']
```

Comment:
Hi @choosehappy I got it from [openslide website](https://github.com/choosehappy/HistoQC/pull/256).

Reply:
got it, thanks. i still think we should move away from this approach as discussed in my github review, did you see it?

Reply:
It's probably not ideal to identify which reader to use by extension, especially when what these readers can handle are not mutually exclusive.

You might want to define a default option if nothing is specified or whatever library requested by the user is not installed, e.g., import these readers by a specified order and use whatever is correctly imported. Otherwise, it should be defined by the user, e.g., put a string or int placeholder in the config file to specify what type of WSIHandle it is.